_data/projects/yes.yml

```diff
@@ -1,8 +1,8 @@
-name: yes
+---
+name: 'yes'
```

Comment:
My favourite change as part of this PR was that `yes` (without the quotes) was being parsed and then emitted as a boolean `true`. Yay YAML :tada: